src/App.js

```diff
@@ -22,7 +23,11 @@ function App() {
           <Route path="/omatpolut" element={<PathSelection />} />
           <Route path="/muokaapolkua/:pathName" element={<ManagePath />} />
           <Route path="/uusisana/:pathName" element={<NewWord />} />
-          <Route path="/peli/:pathName" element={<GamePage />} />
+          <Route path="/peli/:pathName" element={<LevelsPage />} />
+          <Route
+            path="/peli/:pathName/taso/:levelIndex"
```

Comment:
En ihan ymmärtänyt miksi URL:ssa on tason indeksi mukana? Eihän tuolla näytä olevan mitään vaikutusta siihen millä sivulla olen jos olen pelissä? Nyt URL:ssa pomppii nuo indeksit mukana ja nuoli taaksepäin ei tee mitään ennen kuin klikkailee kaikki indeksit alkuun.

Reply:
Tasoja pitäisikin tulla viisi, jos sanoja on kuusi: nyt kaikilla tasoilla on sama määrä sanoja paitsi ensimmäiselle tasolle tulee kaksi sanaa. Jos lisäät seitsemännen sanan myös toiselle tasolle tulee toinen sana. Sanat jakautuvat tasoille tällä logiikka kunnes yhdelle tasolle tulisi enemmän kuin 10 sanaa. Tällöin tulee uudet viisi tasoa, joille käytetään samaa logiikkaa. Tasojen määrä ei ole suoraan sama kuin sanojen määrä, koska käsittääkseni tässä ei ollut maksimimäärää sanoille ja ajattelin, että sadat tai useat kymmenet tasot alkaisi olla liikaa. Toisaalta ymmärsin, että joillain potilailla voi olla tarvetta siihen, että polulla on aika vähän sanoja esim. vaan 10 tai vähemmän. Koitin tähän kehitellä siksi jonkun logiikan, että tasoja olisi järkevä määrä riippumatta sanojen määrästä niin, että pelaaja jolla on vaan 5 sanaa polulla ei joudu pelaamaan yhden tason polkua ja toisaalta tasojen määrä olisi järkevä, vaikka sanoja tulisi 100 tai enemmän. Otin myös huomioon, että yhdellä tasolla olisi enintään 10 sanaa. Voi olla, että tämä menee liian monimutkaiseksi ja täytyy miettiä tuohon joku yksinkertaisempi logiikka sen perusteella kuinka paljon niitä sanoja oikeasti tulee poluille/asetetaanko joku maksimimäärä sanoille? Tein myös niin, että tasojen määrä on nyt viidellä jaollinen paitsi, jos sanoja on vähemmän kuin 5, koska näin voidaan vaan jatkaa sitä polkua tuolla samalla vektorigrafiikalla.  Tämän voisi ehkä koittaa toteuttaa niin, että polku jatkuu aina esim. yhdellä viidesosalla polku-vektorista, kun tarvitaan uusi taso ja uusia tasoja voisi siis lisätä yksitellen. Tässä vaan loppui osaaminen kesken noiden vektorigrafiikoiden kanssa. Tästä koko logiikasta voidaan varmaan keskustellaan tänään lisää, että mikä nyt olisi oikeasti järkevä tapa toteuttaa tämä tasoihin jako..

Reply:
Ikonit tulevat tällä hetkellä randomilla paitsi viimeisellä tasolla on aina se pokaali-ikoni. Tästä voidaan myös tänään keskustella, että miten se halutaan toteuttaa.